pp/event/upload_file.go

```diff
@@ -109,7 +109,11 @@ func RspUploadFile(ctx context.Context, _ core.WriteCloser) {
 		utils.ErrorLog("target.Result is nil")
 
 	} else if target.Result.State != protos.ResultState_RES_SUCCESS {
-		utils.Log("upload failed", target.Result.Msg)
+		if strings.Contains(target.Result.Msg, "Same file with the name") {
```

Comment:
This is very fragile implementation. It is very hard to keep the consistence between SP and PP. We should change it to pass a certain flag. But I will accept it for now.